pkgs/applications/audio/jamulus/default.nix

```
@@ -0,0 +1,30 @@
+{ mkDerivation, stdenv, fetchFromGitHub, pkg-config, qtscript, qmake, libjack2
+}:
+
+mkDerivation {
+  pname = "jamulus";
+  version = "3.5.3";
+  src = fetchFromGitHub {
+    owner = "corrados";
+    repo = "jamulus";
+    rev = "r3_5_3";
```

```suggestion
    rev = "r${stdenv.lib.replaceStrings [ "." ] [ "_" ] version}";
```

also you will need `mkDerivation rec {`